### PR TITLE
Fixed `AcquisitionDuration` for `sub-9605`

### PR DESCRIPTION
**Pull request by alexfoias**

### CODE REVIEW

File: sub-9605/anat/sub-9605_run-2_T2starw.json [hunk 1]
```diff
@@ -3,5 +3,5 @@
 	"MagneticFieldStrength": 7,
 	"Manufacturer": "Siemens",
 	"InstitutionName": "Mount Sinai",
-	"AcquisitionDuration": "167"
-}
\ No newline at end of file
+	"AcquisitionDuration": "507"
```
Comment (jcohenadad) [inline sub-9605/anat/sub-9605_run-2_T2starw.json:6]: why 507?
i calculated from Alan's message: 60*8 + 21=501

**Review by jcohenadad — COMMENTED** [review]
@alexfoias how about run1?

File: sub-9605/anat/sub-9605_run-2_T2starw.json (hunk 1 above)
Comment (alexfoias) [reply sub-9605/anat/sub-9605_run-2_T2starw.json:6]: oups I saw 27s. I will fix it

**Comment by alexfoias** [comment]
Fixed for both runs.

**Review by jcohenadad — APPROVED** [review]

**Pull request merged**